### ISSUE TITLE
NUMP

### ISSUE THREAD

**Issue by maloufabmob** [fabmob/myc-emissions-calculator#19]
## Introduction 

Adding the possibility for a country to calculate emission for a National Urban Mobility Plan

**Comment by maloufabmob** [comment]
- [x] Add the second possibility to describe transport activity (fleet approach), select an approach by default (fleet for NUMP / models for SUMP) but allow the users to choose between the possibilities
- [x] Modify project information page (allowing to choose between a SUMP and a NUMP)

**Comment by maloufabmob** [comment]
_-> based on the excel-based tool_

**Comment by maloufabmob** [comment]
- [ ] Suggesting values from NUMP to SUMP cities that are in the same country (mid-term objective)

**Comment by maloufabmob** [comment]
https://github.com/fabmob/myc-emissions-calculator/issues/20 to come and linked with it

**Comment by TTalex** [comment]
I have added a selection in the project info page. Selecting NUMP removes the City field:
![Image](https://user-images.githubusercontent.com/7023229/183617015-f3d7a360-2611-4388-a134-42f6850ac40c.gif)

I have also modified the transport activity step, I'm unsure about the content and UI, but it works for now.
Vkt are automatically calculated using fleet approach, an additional source also appears when selecting the fleet approach.
![Image](https://user-images.githubusercontent.com/7023229/183616090-980e088c-81ab-4047-b83c-774ef2de61f5.gif)

- [ ] Maybe edit the current text: "Use pre-computed vehicle kilometers travelled information" to be clearer
